docs/faq.md

```diff
@@ -0,0 +1,60 @@
+---
+description: |
+    We have colleted various questions you asked 
+    and combined them with our answers this FAQ.
+---
+
+[Contributing]: https://deno.land/manual@v1.35.1/references/contributing#submitting-a-pr-to-fresh
+[Discussions]: https://github.com/denoland/fresh/discussions
+[Roadmap]: https://github.com/denoland/fresh/issues/563
+[SaaS]: https://deno.com/saaskit
+
+<br>
+
+## Usage
+
+#### What should I **Learn** before using Fresh
+
+HTML, CSS, TypeScript and Preact
+
+<br>
+
+## Project
+
+#### Is there a **Roadmap**
+
+Currently the roadmap is tracked as a **[GitHub Issue][Roadmap]**.
+
+#### Can I talk to the **Core Maintainer**
+
+You can find **Marvin** on the following platforms:
+
+-   Discord **@marvin_h** for quick questions
+
+-   **[GitHub][Discussions]** for longer form discussions
+
+<br>
+
+## Development
+
+#### Is there a **Contribution Guide**
+
+Not one dedicated to Fresh, only the general **[Deno Guidelines][Contributing]**.
+
+#### Do you accept **Localizations** for documentation
+
+We would like to in the future, however it will need to be properly  
+coordinated to keep the translations maintained and up-to-date.
+
+<br>
+
+## Features
+
+#### Do you offer **i18** support
+
+Not yet, it's on the **[Roadmap]**, however not as a top priority.
```

Comment:
This makes it sound `i18n` wouldn't be possible with Fresh. Sure, we don't have some helpers that make it nicer, but any website can add `i18n` support without helper functions from the framework.

Reply:
How about this instead?
![image](https://github.com/denoland/fresh/assets/73050054/3c4ce3d2-98bd-46da-a34b-23457c4130e1)